pkgs/development/libraries/libmikmod/default.nix

```diff
@@ -11,10 +11,10 @@ in stdenv.mkDerivation rec {
   };
 
   buildInputs = [ texinfo ]
-    ++ optionals stdenv.isLinux [ alsaLib libpulseaudio ]
-    ++ optional stdenv.isDarwin CoreAudio;
+    ++ optional stdenv.isLinux alsaLib;
   propagatedBuildInputs =
-    optional stdenv.isLinux libpulseaudio;
+    optional stdenv.isLinux libpulseaudio
+    ++ optional stdenv.isDarwin CoreAudio;
```

Comment:
What does change when you add it to `propgatedBuildInputs`?

Reply:
Since frameworks are impure we should avoid propagating them. It can be a bit verbose and annoying, but I prefer to be more explicit about where we depend on them.

Reply:
👍 Dropped that commit.